internal/util-collection/src/main/scala/sbt/internal/util/Attributes.scala

```diff
@@ -196,11 +196,9 @@ object AttributeMap {
   def apply(entries: AttributeEntry[_]*): AttributeMap = empty ++ entries
 
   /** Presents an `AttributeMap` as a natural transformation. */
-  implicit def toNatTrans(map: AttributeMap): AttributeKey ~> Id = new (AttributeKey ~> Id) {
-    def apply[T](key: AttributeKey[T]): T = map(key)
-  }
-
+  implicit def toNatTrans(map: AttributeMap): AttributeKey ~> Id = λ[AttributeKey ~> Id](map(_))
```

Comment:
I am perfectly happy to go with uber fp polyfunction _value level_ λ notation, but I am really not sure if this increases the "readability."

Reply:
Yeah.. I'm happy to see the reduction in boilerplate, but I'm still not 100% convinced.